### PR TITLE
Update directory-service-accounts.md

### PR DESCRIPTION
**Pull request by martin77s**
minor fixes after Eli's review

### CODE REVIEW

**Comment by PRMerger7** [comment]
@martin77s : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

File: ATPDocs/directory-service-accounts.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ When the sensor starts, it will get a list of DSA entries configured in Defender
 
 #### One DSA entry is configured
 
-The sensor will attempt to use the DSA entry configured during start-up and each time a SAM-R query is made.
+The sensor will attempt to use the DSA entry configured during start-up, as a reaction to a new domain contacting the DC, each time a SAM-R query is made or whenever such a connection needs to be recreated.
```
Comment (dcurwin) [inline ATPDocs/directory-service-accounts.md:61]:
```suggestion
The sensor will attempt to use the DSA entry configured during start-up, as a reaction to a new domain contacting the domain controller, each time a SAM-R query is made, or whenever such a connection needs to be recreated.
```

**Comment by PRMerger10** [comment]
@dcurwin : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: ATPDocs/directory-service-accounts.md [hunk 2]
```diff
@@ -71,9 +71,9 @@ When there are two or more DSA entries, the sensor will try DSA entries in the f
 >[!NOTE]
 >gMSA entries have a higher priority over regular entries.
 
-- The sensor will look for a DSA entry with an exact match of the domain name of the sensor's domain.  If an exact match is found, the sensor will attempt to sign in to the domain with the DSA entry settings.
+- The sensor will look for a DSA entry with an exact match of the domain name of the target domain.  If an exact match is found, the sensor will attempt to sign in to the domain with the DSA entry settings.
 
-- If there isn't an exact match of the domain name or the exact match entry failed to authenticate, the sensor will pick a DSA entry at random.
+- If there isn't an exact match of the domain name or the exact match entry failed to authenticate, the sensor will traverse the list with round robin.
```
Comment (dcurwin) [inline ATPDocs/directory-service-accounts.md:76]:
```suggestion
- If there isn't an exact match of the domain name or the exact match entry failed to authenticate, the sensor will traverse the list via round robin.
```

**Comment by PRMerger10** [comment]
@dcurwin : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by dcurwin — APPROVED** [review]

**Pull request merged**